public/pages/DetectorsList/utils/tableUtils.tsx

```diff
@@ -28,26 +30,99 @@ const renderTime = (time: number) => {
   return DEFAULT_EMPTY_DATA;
 };
 
+const renderLastUpdateTime = (lastUpdateTime: number, detector: Detector) => {
+  const momentTime = moment(detector.lastUpdateTime);
+  if (detector.lastUpdateTime && momentTime.isValid()) {
+    return momentTime.format('MM/DD/YY h:mm A');
```

Comment:
MM/DD/YYYY? According to [mockup](https://aws-uxdr.invisionapp.com/d/?redirHash=#/console/19392952/405140830/comments?scrollOffset=4200)

Reply:
Yes, thanks. Will change